src/test/java/duke/ToDoTest.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ToDoTest {
+    @Test
+    public void testStringConversion() {
+        ToDo task = new ToDo("homework");
+        assertEquals("[T][ ] homework", task.toString());
+    }
+
+    @Test
+    public void testChangeFormat() {
```

Comment:
Underscore can be used in test methods name! e.g. featureUnderTest_testScenario_expectedBehavior() as mentioned in the java coding practice website, perhaps you can try using that format?